Source/WebKit/GPUProcess/graphics/RemoteRenderingBackend.cpp

```
@@ -364,10 +364,17 @@ void RemoteRenderingBackend::cacheFont(const Font::Attributes& fontAttributes, F
     m_remoteResourceCache.cacheFont(WTFMove(font));
 }
 
-void RemoteRenderingBackend::cacheFontCustomPlatformData(Ref<FontCustomPlatformData>&& customPlatformData)
+void RemoteRenderingBackend::cacheFontCustomPlatformData(WebCore::FontCustomPlatformSerializedData&& fontCustomPlatformSerializedData)
 {
     ASSERT(!RunLoop::isMain());
-    m_remoteResourceCache.cacheFontCustomPlatformData(WTFMove(customPlatformData));
+
+    auto customPlatformData = FontCustomPlatformData::tryMakeFromSerializationData(WTFMove(fontCustomPlatformSerializedData), false);
+    if (!customPlatformData.has_value()) {
+        RELEASE_LOG(RemoteLayerBuffers, "[renderingBackend=%" PRIu64 "] RemoteRenderingBackend::cacheFontCustomPlatformData - failed to decode FontCustomPlatformData",  m_renderingBackendIdentifier.toUInt64());
```

This should probably be MESSAGE_CHECK instead of a log and ignoring decode failures.  That'll keep existing behavior.